lucene/core/src/java/org/apache/lucene/codecs/compressing/CompressionMode.java

```diff
@@ -245,6 +340,55 @@ public Decompressor clone() {
       return new DeflateDecompressor();
     }
 
+  }
+  private static final class zstdDecompressor extends Decompressor {
```

Comment:
class name first letter should be upper case

Reply:
okk i will resolve this